go/stats/metrics.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/inverse-inc/packetfence/go/interval"
+	"github.com/inverse-inc/packetfence/go/log"
+	"github.com/inverse-inc/packetfence/go/pfconfigdriver"
+)
+
+var MySQLdatabase *sql.DB
+
+func ProcessMetricConfig(ctx context.Context, conf pfconfigdriver.PfStats) error {
+
+	// Read DB config
+	pfconfigdriver.PfconfigPool.AddStruct(ctx, &pfconfigdriver.Config.PfConf.Database)
+	configDatabase := pfconfigdriver.Config.PfConf.Database
+	connectDB(configDatabase)
+	MySQLdatabase.SetMaxIdleConns(0)
```

Comment:
Lets remove this line since its already being configured in database/connection.go